libra_validation/fssh_based_2_state_models/run_all.py

```diff
@@ -0,0 +1,64 @@
+import os
```

Comment:
what i meant also is - can we be more "compact" and clean, say - let's create manually a list of most interesting methods (say, 20-30 or such), like:


       all_recipes = []

       #   "FSSH" - e.g. select the most appropriate options for the "vanilla FSSH"
       method0 = [0, 0, 1, ...] ;   all_recipes.append(method0)

       #   "GFSH" - e.g. select the most appropriate options for the "vanilla GFSH"
       method1 = [0, 0, 2, ...] ;   all_recipes.append(method1)

       ...